### PR TITLE
Fail fast if no routes set.

### PR DESCRIPTION
**Pull request by apuchitnis**

### CODE REVIEW

File: cmd/winch/main.go [hunk 1]
```diff
@@ -1,3 +1,5 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
```
Comment (bwplotka) [inline cmd/winch/main.go]: To remove (:

File: Gopkg.lock [hunk 2]
```diff
@@ -201,11 +201,11 @@
   revision = "f63a7dfb64c138bd93d5c5b896d8b33c4b08e000"
 
 [[projects]]
-  digest = "1:1acd9ce1ab5ce6fcb48c9b178aa1d6a3cab2e1793997f10c6a937a46fb224db6"
+  digest = "1:8d90bab8af001cc03bb2aac86126cdb76fe0f86b6edd328d3de84bc429ab6ca7"
   name = "github.com/grpc-ecosystem/go-grpc-prometheus"
   packages = ["."]
   pruneopts = ""
-  revision = "39de4380c2e0353a115b80b1c730719c79bfb771"
+  revision = "68e3a13e41175110a447e67246a59873d6aca902"
```
Comment (bwplotka) [inline Gopkg.lock]: Why this updated?

File: cmd/winch/main.go (hunk 1 above)
Comment (apuchitnis) [reply cmd/winch/main.go]: :man_facepalming:

File: Gopkg.lock (hunk 2 above)
Comment (apuchitnis) [reply Gopkg.lock]: fixed

**Pull request merged**